pom.xml

```diff
@@ -530,7 +523,7 @@
                         <groupId>org.apache.james</groupId>
                         <artifactId>apache-mime4j-dom</artifactId>
                     </exclusion>
-                </exclusions>
+                </exclusions>-->
```

Comment:
Is the commented-out code remaining for a reason?

Reply:
no, just forgot to remove it